### PR TITLE
Add store and load functionality

### LINKED ISSUE #19: As an organised home cook, I want to load my saved list
so that I can add on to my existing list

### LINKED ISSUE #22: As an organised home cook, I want to save my list
so that I have a local copy of my list

### PR DESCRIPTION
**Pull request by JLoh579**

### CODE REVIEW

File: src/main/java/seedu/duke/data/ShoppingList.java [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 public class ShoppingList {
 
-    private static ArrayList<Item> items = new ArrayList<>();
+    private ArrayList<Item> items = new ArrayList<>();
```
Comment (chaosInTheWild) [inline src/main/java/seedu/duke/data/ShoppingList.java:10]: Thanks for doing the housekeeping, I agree that this should not be static.

**Review by chaosInTheWild — COMMENTED** [review]
Overall I think the implementing of GSON makes it look cleaner and neater. Thanks for taking the time to understand and implement it too. 

Looks good to merge! :+1:

**Comment by trishaangelica** [comment]
looks good to merge! thanks for introducing GSON to me! (something new to learn) 👍

File: src/main/java/seedu/duke/data/ShoppingList.java (hunk 1 above)
Comment (JLoh579) [reply src/main/java/seedu/duke/data/ShoppingList.java:10]: Np! Actually I think GSON doesn't allow storing the items in the shopping list unless its non-static cause I had an issue with it that was only solved by changing this haha

**Pull request merged**